sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,10 +2,18 @@
 
 ## 7.1.1 (Unreleased)
 
+This version and all future versions will require Python 2.7 or Python 3.6+, Python 3.5 is no longer supported.
```

Comment:
I believe this comment has already been added to [this PR](https://github.com/Azure/azure-sdk-for-python/pull/17751)?

Reply:
will merge first and resolve in your PR :P